tests/unit/checkDeployBlockersTest.ts

```diff
@@ -2,12 +2,19 @@
  * @jest-environment node
  */
 import * as core from '@actions/core';
-import _ from 'underscore';
+import type {Writable} from 'type-fest';
 import run from '../../.github/actions/javascript/checkDeployBlockers/checkDeployBlockers';
 import GithubUtils from '../../.github/libs/GithubUtils';
 
+type Comment = {body: string};
+type Comments = {
```

Comment:
In our guidelines we have a rule to not name types in plural form. please change it